tools/@aws-cdk/remodel/lib/cli.ts

```diff
@@ -72,7 +72,7 @@ export async function main() {
   const templateDir = path.join(__dirname, '..', 'lib', 'template');
   await copyTemplateFiles(templateDir, targetDir);
 
-  await runBuild(targetDir);
+  await cleanup(targetDir);
```

Comment:
We want to keep runBuild as well, right?

Reply:
I want to handle it in a separate task it just complicates things currently